tools/clang/test/HLSLFileCheck/hlsl/objects/WaveMatrix/lit.local.cfg

```diff
@@ -0,0 +1,2 @@
+# HLSL Change - Disable WaveMatrix tests, shader model 6.9 is not enabled yet
+config.unsupported = False
```

Comment:
Seems like unsupported should be `True`?  Or am I reading this wrong?

Reply:
Yes, it should!

Reply:
I was testing this with `hcttest -spirv` and that somehow succeeded, and it did not seem to use lit at all. I'll verify again.

Reply:
Actually, I realized this isn't needed here as these tests aren't run through lit (yet). So we can remove this file entirely. It is harmless though.